api/tests/routes/native/v1/openapi_test.py

```diff
@@ -252,6 +252,12 @@ def test_public_api(client):
                             "title": "Beginningdatetime",
                             "type": "string",
                         },
+                        "features": {
+                            "items": {"type": "string"},
+                            "nullable": True,
```

Comment:
```suggestion
                            "nullable": False,
```
J'imagine que le schéma désiré est ceci :)

Reply:
Yep yep :D